lib/src/migrator.dart

```diff
@@ -143,7 +170,7 @@ class _Migrator extends RecursiveStatementVisitor implements ExpressionVisitor {
       name = builtInFunctionNameChanges[name] ?? name;
       _currentMigration.additionalUseRules.add("sass:$namespace");
     }
-    _currentMigration.patches.add(Patch(node.name.span, "$namespace.$name"));
+    _currentMigration.patches.add(Patch(span, "$namespace.$name"));
```

Comment:
It doesn't actually say this for `get-function()` in the spec yet (oops), but the way [other functions work](https://github.com/sass/language/blob/master/accepted/module-system.md#new-features-for-existing-functions) and the way `get-function()` *should* work is taking an additional optional `$module` parameter that specifies the module, rather than parsing the module name out of the function name.